plugins/quota/actor.go

```diff
@@ -0,0 +1,96 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// isQuotadAffectedNode returns true if there are local bricks of volume on
+// which quota is enabled
+func isQuotadStopRequired(volumes []*volume.Volinfo) bool {
```

Comment:
reverse the return value, 

- return `true` if stop required(No volumes in started state with Quota enabled)
- return `false` if stop not required(Some volumes still running with Quota enabled)

Reply:
done